build/push-ci-build.sh

```diff
@@ -32,5 +32,11 @@ KUBE_GCS_LATEST_CONTENTS=${LATEST}
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "$KUBE_ROOT/build/common.sh"
 
-kube::release::gcs::release
-kube::release::gcs::publish_latest
+MAX_ATTEMPTS=3
+attempt=0
+while [[ ${attempt} -lt ${MAX_ATTEMPTS} ]]; do
+  kube::release::gcs::release && kube::release::gcs::publish_latest && break || true
```

Comment:
though if the "make public" step fails (https://github.com/GoogleCloudPlatform/kubernetes/blob/74b78d537df6dee6842573f2b78ea28dbdf87ade/build/common.sh#L936) there's no way it'll recover correctly, and that happens a fair bit. then we'd potentially "succeed" but the build would be private.